.gitignore

```diff
@@ -1,2 +1,3 @@
 .vscode
 build
+**/vinstr_str.*
```

Comment:
Once you make the shell script to generate the object file and link it to the tilt project, you don't need this to be part of the repository at all.